### PR TITLE
Add Serilog

### PR DESCRIPTION
**Pull request by Blackolf931**

### CODE REVIEW

File: TSP/TSP.API/Program.cs [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+          //  var configuration = new ConfigurationBuilder().AddJsonFile("appsetting.json").Build();
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .Enrich.FromLogContext()
+                .WriteTo.Console()
+                .WriteTo.Debug()
+                .WriteTo.File(@"E:\Innowise\TSP\Serilog\log.json")
```
Comment (nantonov) [inline TSP/TSP.API/Program.cs:19]: Get rid of file

File: TSP/TSP.API/appsettings.json [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
-}
+    "Serilog": {
+      "Using": [],
+      "MinimumLevel": {
+        "Default": "Information",
+        "Microsoft": "Warning",
+        "System": "Warning"
+      },
+      "Enrich": [ "FromLogContext", "WithMachineName", "WithProcessId", "WithThreadId" ],
+      "WriteTo": [
```
Comment (nantonov) [inline TSP/TSP.API/appsettings.json:10]: get rid of seq

**Review by nantonov — DISMISSED** [review]

File: TSP/TSP.API/appsettings.json (hunk 2 above)
Comment (Blackolf931) [reply TSP/TSP.API/appsettings.json]: fix

File: TSP/TSP.API/Program.cs (hunk 1 above)
Comment (Blackolf931) [reply TSP/TSP.API/Program.cs]: fix

**Review by nantonov — APPROVED** [review]

**Pull request merged**